crd-generator/src/main/java/io/strimzi/crdgenerator/annotations/MinimumItems.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.crdgenerator.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
In hindsight documenting the annotations on the CrdGenerator class level Javadoc was a mistake. I know you've documented this there, but I think we should probably document the annotation here too. (In another PR we could move the documentation to each of the annotations we define, and only document the jackson annotations on the CrdGenerator class level Javadoc).

Reply:
`listeners: []` gives `spec.kafka.listeners in the body should have at least 1 items`
I am a little bit confused. Should I change doc in this PR or we will do some larger refactoring later?

Reply:
Perfect! Another PR for the doc is fine.